nettacker/core/messages.py

```
@@ -20,52 +21,51 @@ def application_language():
 
 
 def load_yaml(filename):
-    return yaml.load(StringIO(open(filename, "r").read()), Loader=yaml.FullLoader)
+    with open(filename, "r", encoding="utf-8") as f:
+        return yaml.load(f, Loader=yaml.FullLoader)
```

Ruff (0.13.1)

Probable use of unsafe loader `FullLoader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)